samples/qa-scroller/src/views/MainView.module.less

```diff
@@ -16,7 +16,7 @@ body {
 	display: flex;
 	padding-top: 12px;
 	box-sizing: border-box;
-	border: solid glay 10px;
+	border: solid gray 10px;
```

Comment:
Let's remove the border. I think it looks better.
![image](https://user-images.githubusercontent.com/5037429/141430788-016695c9-26d9-4b34-9beb-05393428617e.png)

Reply:
Done.